CMakeLists.txt

```diff
@@ -202,7 +220,26 @@ add_library(freertos_kernel STATIC
 target_include_directories(freertos_kernel
     PUBLIC
         include
-        ${FREERTOS_CONFIG_FILE_DIRECTORY}
 )
 
-target_link_libraries(freertos_kernel freertos_kernel_port)
+target_compile_options( freertos_kernel
```

Comment:
These do not seem applicable to all the compilers and therefore, should not be here.

Reply:
Disagree - when you have compiler checks for all the various compilers you are supporting you can add the other compilers in here similar to the way they are done here with the ID specified.  This is just Clang and GNU as those were the compilers I had available for my debugging on a linux machine.  This was here to be added to at a later date.

Reply:
They are removed.